odd_collector/adapters/opensearch/client.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+from opensearchpy import OpenSearch
+
+from odd_collector.domain.plugin import OpensearchPlugin
+
+
+class Client:
+    def __init__(self, config: OpensearchPlugin):
+        username, password = config.username, config.password.get_secret_value()
```

Comment:
Be attentive here, if default value for password is `None`, `config.password.get_secret_value()` will cause an error, you can move that statement under 11 line.

Reply:
updated